src/torrent.cpp

```diff
@@ -1094,6 +1094,82 @@ namespace libtorrent
 #endif
 	}
 
+	boost::uint64_t torrent::get_flags() const
```

Comment:
I suppose these functions are closer to ``setf()`` and ``unsetf()``, which they already model pretty closely. It also clearly would be bad to make the setter be blocking just to return the previous flags